src/Components/FruitProduct.js

```diff
@@ -0,0 +1,26 @@
+import ChangeCountButton from "./ChangeCountButton";
+
+
+function FruitProduct({ productName, add, subtract, currentCount}) {
+
+    return (
+        <article>
+            <img src="" alt=""/>
+            <span>{ productName }</span>
+            <ChangeCountButton
```

Comment:
Het zou nog netjes zijn geweest als je deze button zou disablen als de currentCount 0 is